src/coordinator/index.js

```diff
@@ -58,6 +67,69 @@ class Coordinator {
     this.edgeNodes = new EdgeNodes(sdk, request, baseUrl);
   }
 
+  /**
+   * Creates an application to a user's favorited applications
```

Comment:
Can you add some comment about how this only works for Web users using the `auth0WebAuth` session type? (For all of these methods)

It should also probably be noted that these are for the _current_ user, not _a_ user

Reply:
Thanks, I good real English I speak...

By all these methods, do you mean all of contxt coordinator or just the favorite application routes?

Reply:
I think just the favorite application routes

Reply:
I've added a comment warning about the session type requirements to those 3 functions. Will you also double check that the same isn't true about getAllOrganizations() and getAllApplications()?